<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kshitija1108" failures="1" tests="4" name="testNG.MyClass" time="0.023" errors="0" timestamp="2024-01-01T23:46:12 IST" skipped="1">
  <testcase classname="testNG.MyClass" name="a" time="0.004">
    <system-out>
      <![CDATA[TCaexecution is started
Hi a is running
TCais Completed Successfully]]>
    </system-out>
  </testcase> <!-- a -->
  <testcase classname="testNG.MyClass" name="d" time="0.002">
    <system-out>
      <![CDATA[TCdexecution is started
Hi d is running
TCdis Completed Successfully]]>
    </system-out>
  </testcase> <!-- d -->
  <testcase classname="testNG.MyClass" name="c" time="0.000">
    <skipped/>
    <system-out>
      <![CDATA[TCcis Skipped,please check dependent method]]>
    </system-out>
  </testcase> <!-- c -->
  <testcase classname="testNG.MyClass" name="b" time="0.017">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at testNG.MyClass.b(MyClass.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
    <system-out>
      <![CDATA[TCbexecution is started
TCbis failed,please check again]]>
    </system-out>
  </testcase> <!-- b -->
  <system-out>
    <![CDATA[TCbexecution is started
TCbis failed,please check again
TCdexecution is started
Hi d is running
TCdis Completed Successfully
TCaexecution is started
Hi a is running
TCais Completed Successfully
TCcexecution is started
TCcis Skipped,please check dependent method]]>
  </system-out>
</testsuite> <!-- testNG.MyClass -->
